src/page/home/home.tsx

```diff
@@ -71,6 +74,10 @@ function Home() {
           updateTaskListOrder={updateCompletedTaskListOrder}
         />
       </div>
+      <WhatsNewModal
+        isOpen={isModalOpen}
+        onRequestClose={() => setIsModalOpen(false)}
```

Comment:
You can also create onRequestClose in the `useHome.tsx` and import it to user here to avoid in-line functions.

Reply:
Please do study on `in-line` functions and share with me if you find anything as well.